components/layout/RightSideChat.vue

```diff
@@ -82,13 +82,50 @@ const isSelected = (id: number) => {
   return id === Number(route.params.id);
 };
 
-const logger = useLogger('CHAT_HUB::');
 const signal = useSignalR();
 if (!accountStore.accessToken) {
   throw new Error('No access token');
 }
 signal.initialize(accountStore.accessToken);
-signal.registerHandler('UserJoinedChat', (id, dto) => logger.log(id, dto));
+signal.registerHandler('ReceivedChatInvitation', (dto) => {
+  chatStore.addInvitation(dto);
+});
```

Comment:
as there is always same definition for callback and store method we can use shorter syntax with method binding `signal.registerHandler('ReceivedChatInvitation',chatStore.addInvitation)` in every of these registrations